includes/class-newspack-popups-model.php

```diff
@@ -1222,7 +1250,26 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ); ?>"
 			</div>
 			<?php if ( ! $no_overlay_background ) : ?>
 				<div style="opacity: <?php echo floatval( $overlay_opacity ); ?>;background-color:<?php echo esc_attr( $overlay_color ); ?>;" class="newspack-lightbox-shim"></div>
-				</form>
+			<?php endif; ?>
+			<?php if ( 'form_submission' === $dismiss_action ) : ?>
+				<script data-amp-plus-allowed type="text/javascript">
+					var s = document.querySelector('form[data-submit-success]');
+					var d = document.getElementById('<?php echo esc_attr( $element_id . '-close' ); ?>');
+					if ( s && d ) {
+						s.addEventListener('submit',function(e){
+							var observer = new MutationObserver(function(){
```

Comment:
Why is a `MutationObserver` needed here? Shouldn't the `hidden` attribute be removed simply on submission?